README.md

```diff
@@ -69,8 +79,18 @@ The following values are returned in `results.csv`:
 
 #### Mail sending
 
-* `MX Record` - If an MX record was found that contains at least a single mail server.
+* `MX Record` - If an MX record was found that contains at least a
+  single mail server.
 * `Mail Servers` - The list of hosts found in the MX record.
+* `Mail Server Ports Tested` - A list of the ports tested for SMTP and
+  STARTTLS support.
+* `Mail Server Is Listening` - If the mail servers are actually
+  listening on the tested ports.
+* `Mail Server Supports SMTP` - If the mail servers support SMTP.
+* `Mail Server Supports STARTTLS` - If the mail servers support
```

Comment:
Same comments as above. `Mail Server Supports STARTTLS` becomes Boolean (true if only all SMTP-offering endpoints enable STARTTLS`, and `Mail Server Supports STARTTLS Results` listing the results.

Also, same comment that maybe `Domain Supports...` is better language here, too.

Reply:
:+1: